src/ImageGenerators/FileTypes/Video.php

```diff
@@ -23,6 +24,12 @@ public function convert(string $file, Conversion $conversion = null): string
 
         $seconds = $conversion ? $conversion->getExtractVideoFrameAtSecond() : 0;
 
+        $duration = $ffmpeg->getDuration();
+
+        if ($duration < $seconds) {
+            throw InvalidConversionParameter::invalidFrameSecond($seconds, $duration);
```

Comment:
I'm not sure if we should throw an exception here. A more safe default is to get the first frame. If people want an exception instead that could use a custom ImageGenerator